sklearn/model_selection/_search.py

```diff
@@ -555,6 +559,22 @@ def classes_(self):
         self._check_is_fitted("classes_")
         return self.best_estimator_.classes_
 
+    def _generate_candidates(self):
+        """Generates lists of candidates to search as a coroutine
```

Comment:
I use "candidate" to mean a full parameter setting, i.e. multiple parameters and their values.